test/fetchWrapper.js

```diff
@@ -1,21 +1,26 @@
-const fetch = require('../src/fetchWrapper');
+const { fetch, apiError } = require('../src/fetchWrapper');
 const assert = require('assert');
 
-describe('fetchWrapper.js', () => {
-    it('exports a function', () => {
+describe('fetchWrapper', () => {
+    it('exports fetch(), a wrapper around node-fetch', () => {
         assert.equal(typeof fetch, 'function');
     });
 
-    it('emits a custom ApiError on failure', () => {
-        // an empty request will fail
-        fetch()
-        .then(error => {
-            console.log(error);
-            assert(error.response);
-            assert(error.message);
-            assert(error.status);
-            assert(error.toString);
-        })
-        .catch(() => {});
+    it('exports apiError(), a function for throwing a custom error', () => {
+        assert.equal(typeof apiError, 'function');
+    });
+
+    context('fetch()', () => {
+        it('emits a custom apiError on failure', () => {
+            // an empty request will fail
+            fetch()
+            .then(error => {
+                assert(error.response);
+                assert(error.message);
+                assert(error.status);
+                assert(error.toString);
+            })
+            .catch(() => {});
+        });
```

Comment:
Is the answer to this question:

> What's the purpose of exporting `apiError`?

is "so that we can test it properly", I'm gonna argue that this is unnecessary. The first of these two tests ("it exports apiError...") doesn't help at all, and the second of these two tests ("it emits...") should run just fine and be just as valuable without exporting the error object.

Reply:
Yeah, I had attempted to write more unit tests for `apiError` but it didn't work out exactly as I had hoped. Agreed that exporting it is unnecessary.